web/src/reducers/apd.test.js

```diff
@@ -846,28 +844,26 @@ describe('APD reducer', () => {
     it('should add a new activity OKR', () => {
```

Comment:
Might want to update the text describing this test.

Reply:
Oh man, I thought I caught them all. Thanks!